FORMAT.md

```diff
@@ -26,3 +26,17 @@ The overview of format is:
     [(id ...) ...]
 
     (end-group)
+
+## Name Formats
+
+The `<name>` in `(author <name>)` or `(editor <name>)` is a string, its
+interpretation is unspecified.
+
+If the `<name>` is `(name-bibtex <given> <particle> <family> <suffix>)`
```

Comment:
Instead of `name-bibtex`, could use just `bibtex`? It only appears in places where we expect a name, so it should be unambiguous.